ci/github-script/merge.js

```diff
@@ -217,19 +218,20 @@ async function handleMerge({
       }
     }
 
-    const canUseMergeBot = await isMaintainer(comment.user.login)
+    const isMaintainer = await isNixpkgsMaintainer(comment.user.login)
     const isEligible = eligible.has(comment.user.id)
-    const canMerge = result && canUseMergeBot && isEligible
+    const canMerge = result && isMaintainer && isEligible
 
     const body = [
       `<!-- comment: ${comment.node_id} -->`,
+      `@${comment.user.login} wants to merge this PR.`,
       '',
-      'Requirements to merge this PR:',
+      'Requirements to merge this PR with `@NixOS/nixpkgs-merge-bot merge`:',
       ...Object.entries(checklist).map(
         ([msg, res]) => `- :${res ? 'white_check_mark' : 'x'}: ${msg}`,
       ),
-      `- :${canUseMergeBot ? 'white_check_mark' : 'x'}: ${comment.user.login} can use the merge bot.`,
-      `- :${isEligible ? 'white_check_mark' : 'x'}: ${comment.user.login} is eligible to merge changes to the touched packages.`,
+      `- :${isMaintainer ? 'white_check_mark' : 'x'}: ${comment.user.login} is a member of the [nixpkgs-maintainers](https://github.com/orgs/NixOS/teams/nixpkgs-maintainers) team.`,
+      `- :${isEligible ? 'white_check_mark' : 'x'}: ${comment.user.login} is eligible to merge changes to all touched packages.`,
```

Comment:
Should this be more explicit?

```suggestion
      `- :${isEligible ? 'white_check_mark' : 'x'}: ${comment.user.login} is a maintainer of all touched packages.`,
```

Or is it intentionally vague to allow for future changes to eligibility?

Reply:
I can't say whether they are a maintainer of all touched packages, because I am only testing for those in by-name. So once a non-by-name package is in the mix, the proposed change might not reflect reality anymore.

Reply:
So the alternative would be:
```suggestion
      `- :${isEligible ? 'white_check_mark' : 'x'}: ${comment.user.login} is a maintainer of all touched packages in `pkgs/by-name`.`,
```

Which is more explicit, but a little wordy.

Reply:
The refactoring for https://github.com/NixOS/nixpkgs/pull/457652#discussion_r2484681445, also allowed me to only check this condition when all packages are in by-name, thus I went with the first proposal.